CHANGELOG.md

```diff
@@ -8,6 +8,13 @@
 
 [All changes in [[UnreleasedUniFFIVersion]]](https://github.com/mozilla/uniffi-rs/compare/v0.26.0...HEAD).
 
+### What's changed?
+
+- The `rust_future_continuation_callback_set` FFI function was removed.  `rust_future_poll` now
+  inputs the callback pointer.  External bindings authors will need to update their code.
+- `RustBuffer` was changed to use `u64` fields.  This eliminates potential panics with using
```

Comment:
Maybe make this a little more precise - IIUC, it's "when the capacity of the vec exceeds what can be stored in an `i32`" (whereas the "approach" language is talking more about the current impl of vecs).

(all IIUC of course!)